src/locales/de/translations.json

```diff
@@ -61,24 +61,46 @@
       "description": "Wir suchen Entwickler:innen aus Leidenschaft! Schaue Dir unsere offenen Stellen an. Wir freuen uns auf Deine Bewerbung."
     },
     "thank": "Danke für deine Bewerbung!",
-    "email-confirmation": "Wir haben dir eine E-Mail zur Bestätigung geschickt und melden uns bei dir."
+    "email-confirmation": "Wir haben dir eine E-Mail zur Bestätigung geschickt und melden uns bei dir.",
+    "leadbox": {
+      "title": "Deine Stelle nicht dabei?",
+      "subtitle": "Initiativbewerbung",
+      "text": "Georgios Kaleadis, HR Manager",
+      "mail": "georgios.kaleadis@satellytes.com"
```

Comment:
Hm, I don't want to receive applications. We have the following text included already.

> Or send your application to career@satellytes.com

I think we should replace my email with that and remove the text before it.
![Screen Shot 2021-11-08 at 09 44 23](https://user-images.githubusercontent.com/1701755/140710826-8776713a-4228-4cd8-9b16-02f49c69139d.png)

```suggestion
    "leadbox": {
      "title": "Deine Stelle nicht dabei?",
      "subtitle": "Initiativbewerbung",
      "text": "Georgios Kaleadis, HR Manager",
      "mail": "career@satellytes.com"
```